docs/OnTrack/Deployment/Google Cloud/Overview.md

```diff
@@ -14,10 +16,14 @@
 
 ## Overview
 
+The Google Cloud team's main deliverable for the project is to deploy a student-run version of
+OnTrack hosted on Google Cloud Platform (GCP) for the purpose of Thoth Tech having their own
+deployment to develop OnTrack that is separate to Deakin's deployment.
```

Comment:
**suggestion:** This sentence is really long and a bit mouthful. Could we break it up? 🤔 

```suggestion
The Google Cloud team's main deliverable for the project is to deploy a student-run version of
OnTrack which is hosted on the Google Cloud Platform (GCP). The goal is for Thoth Tech to have their own
deployment to develop OnTrack that is separate to Deakin's deployment.
```

Reply:
Fixed in my latest commit. Thanks.